pkgs/development/python-modules/srp/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, buildPythonPackage, fetchPypi, six, lib }:
+
+buildPythonPackage rec {
+  pname = "srp";
+  version = "1.0.15";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "d5b8ed6dc7d3ae2845a16590ef37763bbf15d6049848b85a8c96dfb8a83c984a";
+  };
+
+  propagatedBuildInputs = [ six ];
+
+  # Package doesn't distribute tests.
+  doCheck = false;
```

Comment:
please include tests, pull source from github if needed.

Reply:
I ran the tests, I added a comment. They wouldn't run because it seems that it was using the network for their tests. I had a libssl.so cannot access shared library.